src/rate/bitmax.ts

```diff
@@ -35,7 +35,7 @@ export function makeBitMaxPlugin(opts: EdgeCorePluginOptions): EdgeRatePlugin {
             }
           ]
         } catch (e) {
-          log.warn(`Issue with Bitmax rate data structure ${e}`)
+          log.warn(`Issue with Bitmax rate data structure ${String(e)}`)
```

Comment:
We don't use the rate plugins anymore so we could delete them instead

Reply:
I agree, but I think we need to do this in another PR.